frontend/src/components/Utils/time.test.js

```diff
@@ -120,16 +120,15 @@ test('other time 24h format', () => {
   const actualSingapore = utils.get24hTime(testDate, TZ_SINGAPORE);
   expect(actualCentral).toEqual(expectedCentral);
   expect(actualSingapore).toEqual(expectedSingapore);
-})
+});
 
-test('ISODate empty input tests', () => {  
+test('ISODate empty input tests', () => {
   const testDate = new Date(Date.UTC(2020, 7, 23, 2, 3, 2, 4)).getTime();
   const expected = '2020-08-23';
   expect(utils.getISODate(testDate, '')).toBe(expected);
   expect(utils.getISODate(testDate, null)).toBe(expected);
-})
- })
-})
+});
+
```

Comment:
FYI if you see this change @Hiram-Silvey , the tests were broken so I made this change in this branch. But I am going to open a separate PR so it is not randomly in this branch.